packages/salesforcedx-vscode-core/package.json

```diff
@@ -28,15 +28,17 @@
     "glob": "^7.1.2",
     "moment": "2.20.1",
     "rxjs": "^5.4.1",
+    "sanitize-filename": "^1.6.1",
     "shelljs": "^0.8.1"
   },
   "devDependencies": {
     "@types/adm-zip": "^0.4.31",
     "@types/chai": "^4.0.0",
     "@types/glob": "^5.0.30",
     "@types/mocha": "2.2.38",
-    "@types/node": "^6.0.40",
+    "@types/node": "^7.0.0",
```

Comment:
I would prefer not to mix-and-match `@types/node` version in our packages. I'm not entirely sure of the repercussions of different versions, but it seems safer to be consistent. Thoughts?

Also, is it safe to just go to 7.9.0 to match the node version that Electron ships with?

Reply:
There is no 7.9.0 available of that packages. 7.0.x is the highest 7.x one.

I was thinking of consistent as well but then held back because of the compile issues I noticed just in this module. If we do it global the change might introduce too much noise for the actual change. IMHO it would be better to file a Tech Debt work item and look at this when VS Code is done upgrading to Electron 2.0